.travis.yml

```diff
@@ -64,7 +64,7 @@ script:
   # complexity (--mccabe) and time (--profile, --durations) analysis by pytest
   # are too simplistic and buggy currently
   - python -m pytest --cov --pastebin=failed ;
-  - pprofile $(which py.test) ;
+  # - pprofile $(which py.test) ;
```

Comment:
Could you modify the earlier commit and this commit ?
It's weird to have a commit which adds something and removes it again in the next commit (?)